### ISSUE TITLE
spread and new can not work together in IE8

### ISSUE THREAD

**Issue by caasi** [babel/babel#1612]
`var foobar = new (Foobar.bind());` doesn't run the constructor correctly in IE8.

When the following codes:

```javascript
var foobar = new Foobar(...args);
```

transpile to:

```javascript
"use strict";

var _bind = Function.prototype.bind;

function _toConsumableArray(arr) { if (Array.isArray(arr)) { for (var i = 0, arr2 = Array(arr.length); i < arr.length; i++) arr2[i] = arr[i]; return arr2; } else { return Array.from(arr); } }

var foobar = new (_bind.apply(Foobar, [null].concat(_toConsumableArray(args))))();
```

One will always get the `foobar` as a `{}`, and it breaks codes like [this](https://github.com/goatslacker/alt/blob/v0.16.0/src/alt/utils/StoreUtils.js#L136).

**Comment by sebmck** [comment]
IE8 support is not a priority. There's nothing that could replicate this easily with functions that IE8 supports. You can however use a shim such as [es5-shim](https://github.com/es-shims/es5-shim).

**Comment by caasi** [comment]
Alright. It's my fault.

Thank you, `es5-shim` works well in this case. :-)

**Comment by zloirock** [comment]
It should work without `es5-shim` with `core-js` / `babel/polyfill`. Looks like bug, i'll fix it.

**Comment by zloirock** [comment]
Bug after removing helper.

**Comment by zloirock** [comment]
Should be fixed in `0.9.12`. Please, report about next similar issues in [`core-js`](https://github.com/zloirock/core-js) repo.

**Comment by caasi** [comment]
@zloirock:

> Please, report about next similar issues in [core-js](https://github.com/zloirock/core-js) repo.

Thank you! I will report similar issues in [core-js](https://github.com/zloirock/core-js) repo next time.
